### DISCUSSION TITLE
Does anyone know why s.phmryp.com is showing up on the malware list adguard-malware-shavar? It appears to be used by pandora.com

### DISCUSSION THREAD

**Discussion by aaronccooke** [AdguardTeam/AdGuardHome#5824] [Q&A]
I was trying to find the source of the malware reports seen below and after running Wireshark I was able to narrow is down to the Pandora window I have up in Firefox.  
![image](https://github.com/AdguardTeam/AdGuardHome/assets/36273076/2c3edb52-93bb-4267-861e-d9e8e65fb6ed)
![image](https://github.com/AdguardTeam/AdGuardHome/assets/36273076/e6955f3c-f7ba-42bd-87fb-bcd039e7e9e7)

Here is the Wireshark results...Is this really a malware site (and Pandora is infected) or is this a false positive?  I'm fine with blocking it but I don't want the false positives for malware on my dashboard.
![image](https://github.com/AdguardTeam/AdGuardHome/assets/36273076/3bfbce0e-5720-409e-a13f-573a768faf6b)

**Comment by ghost** [comment]
Please report possible false positives using the instruction [here](https://adguard.com/kb/general/browsing-security/#false-positive-responses).

**Comment by aaronccooke** [comment]
Perfect, thanks.

**Comment by hjmuller** [comment]
I noticed this URL also coming up as "malware" in my instance of Adguard. I reported it via email to the Adguard team. I haven't received a response yet, and the URL continues to be flagged as "malware."
